distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -137,7 +137,7 @@ type Array
     ## Sorts the Array.
 
       Arguments:
-      - comparator: A comparison function that takes two elements and returns
+      - compare_func: A comparison function that takes two elements and returns
```

Comment:
I suppose `comparator` is more readable. The reason why I changed it to `compare_func` was because this PR essentially establishes _comparator_ as a new terminology. Now, _comparator_ is a type that is returned by `Comparable.from x`. But I guess that we don't have to be that strict in the terminology here, especially here it is clear what `comparator` is from the type ascription. I will revert this change.